Mathlib/Combinatorics/SimpleGraph/Basic.lean

```diff
@@ -1772,17 +1714,27 @@ def mapNeighborSet (v : V) : G.neighborSet v ↪ G'.neighborSet (f v)
 
 /-- Given an injective function, there is an embedding from the comapped graph into the original
 graph. -/
-@[simps]
+-- porting note: @[simps] does not work here
```

Comment:
What is the error? Does `@[simps!]` work?

Reply:
I've added to the porting note.

Floris said `simps` doesn't actually support this case, and it only supported it in Lean 3 because the `{... with ...}` notation would fill in each field individually. Like this works:
```lean
@[simps]
protected def comap (f : V ↪ W) (G : SimpleGraph W) : G.comap f ↪g G :=
  { toFun := f, inj' := f.inj', map_rel_iff' := by simp }
```
He said we shouldn't write it this way though.

It sounds like Floris might get around to figuring out how to support this properly once some open simps PRs get merged.